monitoring/pagerduty_alert.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+
+module: pagerduty_alert
+short_description: Trigger, acknowledge or resolve PagerDuty incidents
+description:
+    - This module will let you trigger, acknowledge or resolve a PagerDuty incident by sending events
+version_added: "1.9"
+author:
+    - "Amanpreet Singh (@aps-sids)"
+requirements:
+    - PagerDuty API access
+options:
+    service_key:
+        description:
+            - The GUID of one of your "Generic API" services.
+            - This is the "service key" listed on a Generic API's service detail page.
+        required: true
+    event_type:
```

Comment:
A more imperative language here might be more inline with anisble modules:

state: triggered, acknowledged, resolved

also check if the incident is already in that state to report on changed correctly